tests/network/primary_pod_network.go

```diff
@@ -89,21 +90,31 @@ var _ = SIGDescribe("[rfe_id:694][crit:medium][vendor:cnv-qe@redhat.com][level:c
 	Describe("Connectivity", func() {
 		const servicePort = 1500
 		const ipv4HeaderSize = 20
+		const ipv6HeaderSize = 40
 		const icmpHeaderSize = 8
-		const ipv4IcmpHeaderSize = ipv4HeaderSize + icmpHeaderSize
 
-		AssertConnectivity := func(outboundVMI, inboundVMI *v1.VirtualMachineInstance) {
-			inboundVMIAddress := inboundVMI.Status.Interfaces[0].IP
-
-			By("Checking connectivity via ICMP")
-			Expect(tests.PingFromVMConsole(outboundVMI, inboundVMIAddress)).To(Succeed())
-
-			By("Checking connectivity via ICMP with MTU-sized frames")
-			mtu := getPodNetworkMTU(outboundVMI)
-			Expect(tests.PingFromVMConsole(outboundVMI, inboundVMIAddress, "-c5", "-w5", "-s", strconv.Itoa(mtu-ipv4IcmpHeaderSize))).To(Succeed())
-
-			By("Checking connectivity via HTTP")
-			Expect(tests.PingHTTPFromVMConsole(outboundVMI, inboundVMIAddress, servicePort)).To(Succeed())
+		AssertConnectivity := func(outboundVMI, inboundVMI *v1.VirtualMachineInstance, checkIPv6 bool) {
```

Comment:
Following the recent talks on dual-stack epic regarding separate tests to be ipv4 and ipv6, do we maybe want to split this function to 2?

Reply:
I dropped the dual-stack check, waiting for Alona's https://github.com/kubevirt/kubevirt/pull/4156 to land. I'd prefer to get this in as is for the sake of conformance tests.